src/kernel/panic.zig

```diff
@@ -309,10 +360,16 @@ test "parseAddr" {
 
 test "parseAddr fails" {
     // Test without terminating whitespace, should fail
-    const str: []const u8 = "1a2b3c4d";
-    const end = @ptrCast(*const u8, str.ptr + 9);
+    var str: []const u8 = "1a2b3c4d";
+    var end = @ptrCast(*const u8, str.ptr + 9);
     var ptr = str.ptr;
     testing.expectError(PanicError.InvalidSymbolFile, parseAddr(&ptr, end));
+
+    // Test with invalid hexdecimal integer
```

Comment:
Might be better to have this as a separate test and the name will convey what this is testing. Then can keep the `const` variables.